### PR TITLE
added esConsumes to modules in EventFilter/HcalRawToDigi

### PR DESCRIPTION
**Pull request by JamminJones**
#### PR description:

added esConsumes to 4 modules

#### PR validation:

the code compiles

### CODE REVIEW

**Comment by Dr15Jones** [comment]
This change was made under my supervision

**Comment by Dr15Jones** [comment]
please test

File: EventFilter/HcalRawToDigi/plugins/HcalDigiToRawuHTR.cc [hunk 1]
```diff
@@ -76,6 +78,8 @@ HcalDigiToRawuHTR::HcalDigiToRawuHTR(const edm::ParameterSet& iConfig)
       tok_HBHEDigiCollection_(consumes<HBHEDigiCollection>(iConfig.getParameter<edm::InputTag>("HBHEqie8"))),
       tok_HFDigiCollection_(consumes<HFDigiCollection>(iConfig.getParameter<edm::InputTag>("HFqie8"))),
       tok_TPDigiCollection_(consumes<HcalTrigPrimDigiCollection>(iConfig.getParameter<edm::InputTag>("TP"))),
+      tok_electronicsMap_(
+          esConsumes<HcalElectronicsMap, HcalElectronicsMapRcd>(edm::ESInputTag("", "electronicsMapLabel_"))),
```
Comment (Dr15Jones) [inline EventFilter/HcalRawToDigi/plugins/HcalDigiToRawuHTR.cc:82]:
```suggestion
          esConsumes<HcalElectronicsMap, HcalElectronicsMapRcd>(edm::ESInputTag("", electronicsMapLabel_))),
```
The value is in the variable, not a string itself.

File: EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc [hunk 2]
```diff
@@ -35,6 +33,9 @@ HcalRawToDigi::HcalRawToDigi(edm::ParameterSet const& conf)
       expectedOrbitMessageTime_(conf.getUntrackedParameter<int>("ExpectedOrbitMessageTime", -1)) {
   electronicsMapLabel_ = conf.getParameter<std::string>("ElectronicsMap");
   tok_data_ = consumes<FEDRawDataCollection>(conf.getParameter<edm::InputTag>("InputLabel"));
+  tok_dbService_ = esConsumes<HcalDbService, HcalDbRecord>();
+  tok_electronicsMap_ =
+      esConsumes<HcalElectronicsMap, HcalElectronicsMapRcd>(edm::ESInputTag("", "electronicsMapLabel_"));
```
Comment (Dr15Jones) [inline EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc:38]:
```suggestion
      esConsumes<HcalElectronicsMap, HcalElectronicsMapRcd>(edm::ESInputTag("", electronicsMapLabel_));
```

**Comment by perrotta** [comment]
+1
- Technical
- jenkins tests pass

**Comment by silviodonato** [comment]
+1

**Pull request merged**